ops/storage-ddls/cassandra-setup.cql

```diff
@@ -231,8 +248,8 @@ CREATE TABLE events(
     topics frozen<set<text>>,
     placeids frozen<set<text>>,
     fulltext text, /* conjunction of title and body to enable querying of both at the same time */
-    PRIMARY KEY (eventid)
-);
+    PRIMARY KEY (eventid, eventtime)
+) WITH CLUSTERING ORDER BY (eventtime DESC);
```

Comment:
the natural ordering of events should always be by `eventtime`. We could manage the sorting in the dashboard, but that could get hairy with paging.